packages/@sanity/core/src/actions/dataset/listDatasetCopyJobs.js

```diff
@@ -0,0 +1,82 @@
+module.exports = async function listDatasetCopyJobs(args, context) {
+  const {apiClient, output, chalk} = context
+  const flags = args.extOptions
+  const client = apiClient()
+  const projectId = client.config().projectId
+  const query = {}
+  let response
+
+  if (flags.offset && flags.offset >= 0) {
+    query.offset = flags.offset
+  }
+  if (flags.limit && flags.limit > 0) {
+    query.limit = flags.limit
+  }
+
+  try {
+    response = await client.request({
+      method: 'GET',
+      uri: `/projects/${projectId}/datasets/copy`,
+      query,
+    })
+  } catch (error) {
+    if (error.statusCode) {
+      output.print(`${chalk.red(`Dataset copy list failed:\n${error.response.body.message}`)}\n`)
+    } else {
+      output.print(`${chalk.red(`Dataset copy list failed:\n${error.message}`)}\n`)
+    }
+  }
+
+  if (response && response.length > 0) {
+    output.print('Dataset copy jobs for this project:')
+    const print = []
+
+    response.forEach((job) => {
+      const {id, state, createdAt, updatedAt, sourceDataset, targetDataset, withHistory} = job
+
+      const createdAtInSeconds = new Date(createdAt).getTime() / 1000
+      const nowInSeconds = new Date().getTime() / 1000
+      const timePassedInSeconds = nowInSeconds - createdAtInSeconds
+      const timeStarted = convertTimestampToDaysHoursMinsSecs(timePassedInSeconds)
+
+      const updatedAtInSeconds = new Date(updatedAt).getTime() / 1000
+      const timeTakenInSeconds = updatedAtInSeconds - createdAtInSeconds
+      const timeTaken = convertTimestampToDaysHoursMinsSecs(timeTakenInSeconds)
+
+      print.push({
+        jobId: id,
+        state: state,
+        history: withHistory,
+        'time started': `${timeStarted} ago`,
+        'time taken': timeTaken,
+        'source dataset': sourceDataset,
+        'target dataset': targetDataset,
+      })
+    })
+
+    output.printTable(print)
+  } else {
+    output.print("This project doesn't have any dataset copy jobs")
+  }
+}
+
+function convertTimestampToDaysHoursMinsSecs(timePassedInSeconds) {
```

Comment:
`sanity` already imports `Moment.js`, should we use that?

Reply:
It seems that Moment.js is considered a legacy project - https://www.npmjs.com/package/moment

I tried it out and it rounds up the time according to some internal rules. See https://momentjs.com/docs/#/displaying/fromnow/
Do we care if it says "10 hours ago" instead of "9 hours, 20 minutes, 44 seconds ago"? 🤔 

With `Moment.js`:
<img width="127" alt="Screenshot 2022-05-23 at 17 40 34" src="https://user-images.githubusercontent.com/24877689/169856731-0dc54364-6ae6-45a2-a0c6-37394f2bb058.png">


With custom code:
<img width="325" alt="Screenshot 2022-05-23 at 17 41 03" src="https://user-images.githubusercontent.com/24877689/169856805-c28e2891-5256-48f5-81c5-d2c5ee916d13.png">

Reply:
We already have some code available when it comes to reading time in `base` -> `useTimeAgo`. Though since it's a hook I don't think we could use it here anyway 🤔 

I don't know if hours are valuable here but my gut feeling tells me that we should use moment instead of adding more logic that I don't feel would add a lot? But I might be missing context and the hours make sense. Thoughts? @rexxars @skogsmaskin

Reply:
I agree with Rita here. If we are already using Moment then it being legacy isn't as big an issue as if we were adding a new dependency. 

I would rather we display "10 hours ago" instead of "9 hours, 20 minutes, 44 seconds ago". In this context, the exact time is not relevant, i.e. I don't care about the seconds or minutes when the time was weeks ago. If they want exact timing then use the HTTP API.

Reply:
Thanks all! I removed the custom code and used `date-fns` as suggested. The result:
<img width="267" alt="Screenshot 2022-06-02 at 12 02 19" src="https://user-images.githubusercontent.com/24877689/171607243-8eb04a48-10d4-45ca-ae6e-9cc425be0086.png">